project-1/main.py

```diff
@@ -0,0 +1,193 @@
+# pylint: disable=import-error
+from pathlib import Path
+from google.oauth2 import service_account
+from gspread_dataframe import set_with_dataframe
+from exceptions import *
+import gspread
+import pandas as pd
+import logging
+import subprocess
+
+
+# logging config
+logging.basicConfig(
+    level=logging.INFO,
+    format="%(asctime)s [%(levelname)s] %(message)s",
+    handlers=[logging.StreamHandler()],
+)
+
+CREDENTIALS_PATH = "./credentials.json"
```

Comment:
Nice, we could just move this entirely to the environment as variables, so this makes it very easy to change these values and not bother changing the code in the future, sorry if I didn't make that clear before

Reply:
quite true chief ... I'd make changes